content/blog/2017/2017-03-12-toulousejam-pipeline-workshop.adoc

```diff
@@ -0,0 +1,117 @@
+---
+layout: post
+title: "Pipeline Workshop & Hackergarten @ ToulouseJAM Feedback"
+tags:
+- hackergarten
+- jam
+- meetup
+- feedback
+author: batmat
+imagesdir: /images/post-images/2017-02-toulousejam-workshop
```

Comment:
`:imagesdir` is an AsciiDoc attribute, it won't work if placed in the YAML front-matter.

Best way to think of it, think of YAML front matter as the document's meta-data. It tells the rendering engine about the file, but it's wholly separate from the content (AsciiDoc)